ds-caselaw-ingester/lambda_function.py

```diff
@@ -189,12 +189,16 @@ def get_consignment_reference(message):
 
 def extract_docx_filename(metadata: dict, consignment_reference: str) -> str:
     try:
-        return metadata["parameters"]["TRE"]["payload"]["filename"]
+        docx_filename = metadata["parameters"]["TRE"]["payload"]["filename"]
     except KeyError:
         raise DocxFilenameNotFoundException(
             f"No .docx filename was found in metadata. Consignment Ref: {consignment_reference}"
         )
 
+    # 2023-08-23: data/ was erroneously added to the v2 filename. This should go away soon.
```

Comment:
where was the added to? To the example doc we are sending to parse to tre v2, or TRE have made a change on their side?

Reply:
ah sorry I misread this as `2023-08-23: data/` was added to the v2 filename.

I remember now that `data/` part that was added by tre for their v2 tar but we have spoken with them and they are reverting that

Reply:
In the v2 metadata.json, the parameters.TRE.filename is "data/myfile.docx" (I think I'm missing a step on the path there)